tests/bitops_test.py

```diff
@@ -13,82 +13,70 @@ def setUp(self):
     def test_popcount(self):
         # Method invocation
         # Toy input
-        p = self.a.popcount()
-        ans = ak.array([0, 1, 1, 2, 1, 2, 2, 3, 1, 2])
-        self.assertTrue((p == ans).all())
+        ans = [0, 1, 1, 2, 1, 2, 2, 3, 1, 2]
+        self.assertListEqual(self.a.popcount().to_ndarray().tolist(), ans)
 
         # Test uint case
-        p = self.b.popcount()
-        ans = ak.cast(ans, ak.uint64)
-        self.assertTrue((p == ans).all())
+        ans = ak.array(ans, ak.uint64)
+        self.assertListEqual(self.b.popcount().to_ndarray().tolist(), ans.to_ndarray().tolist())
```

Comment:
Not a huge thing, we could update this to use `numpy` and remove the `.to_ndarray()` call.

```python
ans = np.array(ans, np.uint64)
self.assertListEqual(self.b.popcount().to_ndarray().tolist(), ans.tolist())
```
This is not required by any stretch, but thought I would mention since we were simplifying things in other cases.